engine/modelc/src/modelimporter_jni.cpp

```diff
@@ -65,14 +148,14 @@ static jobjectArray CreateMaterialsArray(JNIEnv* env, dmModelImporter::jni::Type
     for (uint32_t i = 0; i < count; ++i)
     {
         const dmModelImporter::Material* material = &materials[i];
-        jobject obj = CreateMaterial(env, types, material);
+        jobject obj = C2J_CreateMaterial(env, types, material);
```

Comment:
Here I chose to _not_ use the object caching mechanism, as these structs are so many to manually maintain.
And they should also be small enough, and not as reused so it shouldn't be a huge deal.

The caveat with this type function is, until I've fixed the object cache feature, is that you cannot do it with cyclic dependencies, as it doesn't know when to sto creating new objects.
Once fixed, we could potentially create the entire scene with just one generated call.